SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -39,9 +44,14 @@ void USpatialGDKSettings::PostEditChangeProperty(FPropertyChangedEvent& Property
 
 	if (PropertyName == GET_MEMBER_NAME_CHECKED(USpatialGDKSettings, bUsingQBI))
 	{
-		FMessageDialog::Open(EAppMsgType::Ok,
-			FText::FromString(FString::Printf(TEXT("If you are not using auto-generated launch config, you must make sure to set the value of the \"enable_chunk_interest\" field to \"%s\" in your launch configuration for this to work. (You can check what launch configuration you are using in the SpatialOS GDK for Unreal Editor Settings.)"),
+		const EAppReturnType::Type Result = FMessageDialog::Open(EAppMsgType::YesNo,
+			FText::FromString(FString::Printf(TEXT("You must make sure to set the value of the \"enable_chunk_interest\" Legacy flag to \"%s\" in your launch configuration for this to work.\n\nIf you are using auto-generated launch config, this can be changed from the SpatialOS GDK for Unreal - Editor Settings\n\nDo you want to configure your launch config settings now?"),
```

Comment:
```suggestion
			FText::FromString(FString::Printf(TEXT("You must set the value of the \"enable_chunk_interest\" Legacy flag to \"%s\" in your launch configuration file for this to work.\n\nIf you are using an auto-generated launch config, you can set this value from within Unreal Editor by going to Edit > Project Settings > SpatialOS GDK for Unreal > Settings.\n\nDo you want to configure your launch config settings now?"),
```
LMK if I've misunderstood where you go to set this value, or if you think it's OTT to put the full instructions in here.